### ISSUE TITLE
Aliucord's Http Exception Error

### ISSUE THREAD

**Issue by TheGoldenSun** [Aliucord/Aliucord#264]
### Discord Account

Fukui#6495

### What happens when the bug or crash occurs?

I had been using aliucord version 126.18 (stable) for a long time until I discovered an updated version 126.21 I tried two different WiFi networks, cleared the cache and reinstalled the aliucord installer version 1.0.7 and 1.0.6 (downgrade) to see if it would patch up regular discord's stable version app into an apk file for aliucord, nothing worked is there an explanation for this? because now I can't reinstall aliucord version 126.18 (stable) on my Galaxy a21 device, please fix.

### What is the expected behaviour?

Improve http exception connection from closing secured WiFi networks for Galaxy a21.

### How do you recreate this bug or crash?

![Screenshot_20220829_185641](https://user-images.githubusercontent.com/104534608/187317257-aa468fe8-b638-484b-ad7f-47d290e3cdbe.png)

### Crash-log

```
Checking cached apk..
Downloading discord apk..

An exception occurred while downloading Discord. Please try again. If this error persists, try a different network.
HttpException: Connection closed while receiving data, uri = https://aliucord.com/download/direct/126021/base.apk

```

### Request Agreement

- [X] I did indeed check to make sure the bug or crash report is applicable!

**Comment by Vendicated** [comment]
This is an issue with your connection. Try a different network or download the apk manually, enable devmode and use it from the file system

**Comment by TheGoldenSun** [comment]
I used a different Wi-Fi network on my Galaxy a21 device, what else you want me to try? 





  


Closed #264 as completed.

**Comment by DiamondMiner88** [comment]
It's up to you at this point since it's not an issue from our side. If your ISP blocks aptoide/aliucord.com as part of a safety mechanism that might be the issue
